src/main/java/org/candlepin/subscriptions/metering/MeteringConfiguration.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2009 - 2019 Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.subscriptions.metering;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Top level config class for metering. Beans are defined/configured
+ * on a profile basis via the profile package. This class serves as
+ * an application entry point for the metering components.
+ */
+@Configuration
+@ComponentScan({"org.candlepin.subscriptions.metering.profile"})
```

Comment:
Why not just scan `org.candlepin.subscriptions.metering` here?

Reply:
@kahowell I did it this way because I wanted the profile itself to dictate what to include. Chasing around what beans get automatically loaded, and in what circumstances, is a PITA sometimes.

If you feel strongly about this, let me know and I'll make the change.

Reply:
I was just curious. Given we plan to adjust our pattern around beans in the future, not worried about it.